core/spam/statistics.go

```diff
@@ -0,0 +1,23 @@
+package spam
+
+type Statistics struct {
+	Proposals         Statistic
+	Delegations       Statistic
+	Transfers         Statistic
+	NodeAnnouncements Statistic
+	Votes             []VoteStatistic
+}
+
+type VoteStatistic struct {
+	Proposal    string
+	Total       string
```

Comment:
why are those `strings` rather than uint64?

Reply:
It was because in a lot of other places we store numbers as strings, and in the back of my mind, I've somehow made it a rule. I will change it.